bash/curl/security/curl-pipe-bash.yaml

```diff
@@ -25,9 +25,11 @@ rules:
     - audit
     likelihood: LOW
     impact: LOW
-  patterns:
```

Comment:
@JacobBrackett you have an indentation error here - the `patterns` top level key is now a child key of `metadata` and semgrep cannot find a valid pattern.